diff --git a/configure.ac b/configure.ac
index 0762dfe..7762d38 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,4 +1,4 @@
-AC_INIT([monodevelop], m4_esyscmd_s([grep ^Version ../version.config|cut -d '=' -f 2|tr -d '\n']), [monodevelop-list@lists.ximian.com])
+AC_INIT([monodevelop], m4_esyscmd_s([grep ^Version version.config|cut -d '=' -f 2|tr -d '\n']), [monodevelop-list@lists.ximian.com])
 AC_PREREQ(2.53)
 AM_INIT_AUTOMAKE([1.10 tar-ustar])
 AM_MAINTAINER_MODE
@@ -12,9 +12,9 @@ ASSEMBLY_VERSION=4.0.0.0
 # the C# side of things. It should be one of the following two formats:
 # 1) "VERSION_NUMBER"					"2.0"
 # 2) "VERSION_NUMBER BUILD_TYPE BUILD_NUMBER"		"2.0 Alpha 1"
-PACKAGE_VERSION_LABEL="m4_esyscmd_s([grep ^Label ../version.config|cut -d "=" -f 2|tr -d '\n'])"
+PACKAGE_VERSION_LABEL="m4_esyscmd_s([grep ^Label version.config|cut -d "=" -f 2|tr -d '\n'])"
 
-COMPAT_ADDIN_VERSION=m4_esyscmd_s([grep ^CompatVersion ../version.config|cut -d "=" -f 2|tr -d '\n'])
+COMPAT_ADDIN_VERSION=m4_esyscmd_s([grep ^CompatVersion version.config|cut -d "=" -f 2|tr -d '\n'])
 
 AC_PATH_PROG(MONO, mono)
 AC_PATH_PROG(MCS, mcs)
